teuthology/suite/run.py

```diff
@@ -516,13 +520,45 @@ def schedule_suite(self):
         ).name
         self.base_yaml_paths.insert(0, base_yaml_path)
 
+        # compute job limit in respect of --sleep-before-teardown
+        job_limit = self.args.limit or 0
+        sleep_before_teardown = int(self.args.sleep_before_teardown or 0)
+        if sleep_before_teardown:
+            if job_limit == 0:
+                log.warning('The --sleep-before-teardown is provided, '
```

Comment:
Please change to `The --sleep-before-teardown option was provided: only 1 job will be . . .`

- insert the word "option"
- s/is/was/
- s/,/:/

Reply:
aye aye